wordle-web/components/player-info.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-card justify-center class="p-6">
+    <v-card>
+      <v-btn id="playerName" max-width="128" @click="toggleDialog">{{
+        getName()
```

Comment:
This should update as the player types and changes their name. You can do this by referencing the variable the v-text-field v-model references (playerName) as discussed in class.